dotnet/src/SemanticKernel.Test/AI/OpenAI/Services/AzureOpenAIConfigTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Tests are under the SemanticKernel.UnitTest project now. Addressing the project scoping should reduce some added files in this PR

Reply:
Refresh should have this right.